pythermalcomfort/models/bfu_occupational.py

```diff
@@ -0,0 +1,323 @@
+    speed : float or list of floats, optional
+        Worker speed, [m/s]. Defaults to 1.66.
+    fixed_sweat_rate_lph : float or list of floats, optional
+        Fixed sweat production rate, [L/h]. If ``None`` the empirical humidity function defined by Foster is used. Defaults to ``1.0``.
+    body_surface_area : float or list of floats, optional
+        Body surface area, [m²]. Defaults to 1.81.
+    round_output : bool, optional
+        If ``True`` numeric outputs are rounded (storage components to one decimal, efficiencies to two decimals). Defaults to ``True``.
+
+    Returns
+    -------
+    BFU_occupational
+        A dataclass containing fan-on / fan-off storage rates, their components, and
+        an interpretation code where ``1`` indicates beneficial fan use, ``0`` neutral,
+        and ``-1`` harmful.
+    """
+    inputs = BFU_occupational_Inputs(
+        tdb=tdb,
+        rh=rh,
+        tr=tr,
+        clo=clo,
+        met=met,
+        fan_velocity=fan_velocity,
+        fan_off_velocity=fan_off_velocity,
+        position=position,
+        activity=activity,
+        speed=speed,
+        fixed_sweat_rate_lph=fixed_sweat_rate_lph,
+        body_surface_area=body_surface_area,
+        round_output=round_output,
+    )
+
+    tdb_arr = np.asarray(inputs.tdb, dtype=float)
+    tr_arr = np.asarray(inputs.tr, dtype=float)
+    rh_arr = np.asarray(inputs.rh, dtype=float)
+    clo_arr = np.asarray(inputs.clo, dtype=float)
+    met_arr = np.asarray(inputs.met, dtype=float)
+    bsa_arr = np.asarray(inputs.body_surface_area, dtype=float)
+    speed_arr = np.asarray(inputs.speed, dtype=float)
+    fan_on_arr = np.asarray(inputs.fan_velocity, dtype=float)
+    fan_off_arr = np.asarray(inputs.fan_off_velocity, dtype=float)
+
+    (
+        tdb_arr,
+        tr_arr,
+        rh_arr,
+        clo_arr,
+        met_arr,
+        bsa_arr,
+        speed_arr,
+        fan_on_arr,
+        fan_off_arr,
+    ) = np.broadcast_arrays(
+        tdb_arr,
+        tr_arr,
+        rh_arr,
+        clo_arr,
+        met_arr,
+        bsa_arr,
+        speed_arr,
+        fan_on_arr,
+        fan_off_arr,
+    )
+
+    shape = tdb_arr.shape
+
+    fixed_sweat = inputs.fixed_sweat_rate_lph
+    if fixed_sweat is not None:
+        fixed_sweat_arr = np.broadcast_to(
+            np.asarray(fixed_sweat, dtype=float),
+            shape,
+        )
+    else:
+        fixed_sweat_arr = None
+
+    position_arr = np.broadcast_to(
+        np.asarray(inputs.position, dtype=object),
+        shape,
+    )
+    activity_arr = np.broadcast_to(
+        np.asarray(inputs.activity, dtype=object),
+        shape,
+    )
+
+    is_seated = position_arr == "seated"
+    is_walk = activity_arr == "walk"
+    is_rest = activity_arr == "rest"
+    effective_speed = np.where(is_rest, 0.0, speed_arr)
+
+    # Base physiological and physical properties
+    vo = np.where(is_seated, 0.07, 0.11)
```

Comment:
we should try to use the same variable names as in the other model, if the variable has not been previously defined we should add an inline comment and explain what it is.